test/controllers/api/auth/episodes_controller_test.rb

```diff
@@ -146,6 +146,95 @@ class << @controller; attr_accessor :prx_auth_token; end
       assert_equal episode_update.reload.contents.with_deleted.size, 1
       assert_equal episode_update.contents.with_deleted.first.position, 1
     end
+
+    it "can add uncut to an episode" do
+      update_hash = {
+        uncut: {
+          href: "https://s3.amazonaws.com/prx-testing/test/change1.mp3"
+        }
+      }
+
+      assert_equal episode_update.contents.size, 0
+      assert_nil episode_update.uncut
+      assert_nil episode_update.medium
+
+      @controller.stub(:process_media, true) do
+        put(:update, body: update_hash.to_json, as: :json,
+          params: {id: episode_update.guid, api_version: "v1", format: "json"})
+      end
+      assert_response :success
+
+      episode_update.reload
+
+      contents = episode_update.contents
+      assert_equal contents.size, 0
+
+      uncut = episode_update.uncut
+      assert_not_nil uncut
+      assert_equal "uncut", episode_update.medium
+
+      # put to update without setting should not delete uncut
```

Comment:
This test below shows that when only title is sent in the body on a put, it does not then delete uncut.